src/MessagingBubble/MessagingBubble.tsx

```diff
@@ -53,7 +57,19 @@ export const MessagingBubble: React.FC<Props> = ({
           {children}
         </Linkify>
       </div>
-      {attachments && attachments}
-    </>
+      {attachments && (
+        <div
+          className={cx(
+            cssClasses.MESSAGE_BASE,
+            cssClasses.FLEXBOX,
```

Comment:
Is there a reason you added a `.flexbox` class instead of using the `<FlexBox>` component?